3545/1/points_calculator.rb

```diff
@@ -0,0 +1,34 @@
+class PointsCalculator
+  def initialize(gem_parameters)
+    @gem_parameters = gem_parameters
+  end
+
+  def calculate_stars
+    @gem_parameters[1] * 20
+  end
+
+  def calculate_watchers
+    @gem_parameters[2] * 200
+  end
+
+  def calculate_forks
+    @gem_parameters[3] * 50
+  end
+
+  def calculate_contributors
+    @gem_parameters[4] * 300
+  end
+
+  def calculate_issues
+    @gem_parameters[5] * -2
+  end
+
+  def calculate_users
+    @gem_parameters[6] * 2
+  end
+
+  def calculate_all
```

Comment:
what about making this in one line?

Reply:
Just like as in pack_all method where you have noticed the same thing, hound just doesn't accept such long lines, so I have divided it. Is it acceptable to avoid it like I did?